players/computer.py

```diff
@@ -1,9 +1,50 @@
-import random
+import copy
 from players.player import Player
+from game_rules import GameRules
 
 class Computer(Player):
     def __init__(self, marker):
         Player.__init__(self, marker)
+        self.rules = GameRules()
 
     def choose_move(self, board):
-        return random.choice(board.available_spots())
+        opponent = self.next_player(self.marker)
+        self.best_move(board, opponent)
+        return self.best_move_var
+
+    def move_score(self, board, last_move, depth):
+        if self.rules.win(board, last_move) and last_move == self.marker:
+            return 10 - depth
+        if self.rules.win(board, last_move) and last_move != self.marker:
+            return depth - 10
+        if self.rules.tie(board):
+            return 0
+
+    def next_player(self, last_move):
+        if last_move == 'X':
+            return 'O'
+        else:
+            return 'X'
```

Comment:
This is very similar to Player#define_marker. Is there a better place where we can refactor this logic to?

Reply:
resolved in https://github.com/olgaboiar/ttt_python/pull/7/commits/ecb2dd4599436c76dcdc9a9c6c19fe5e8b18c466
- removed this method from Computer class, because the logic is the same as in parent Player class method
- renamed Player class define_marker method into switch_marker
- replaced all occurrences of define_marker with switch_marker